### PR TITLE
2198 breakup hvac swig

### PR DESCRIPTION
**Pull request by kbenne**

### CODE REVIEW

**Comment by macumber** [comment]
FYI https://github.com/NREL/OpenStudio/issues/3012

File: openstudiocore/src/model/CMakeLists.txt [hunk 1]
```diff
@@ -2015,6 +2015,16 @@ MAKE_SWIG_TARGET(OpenStudioModelGeometry model "${CMAKE_CURRENT_SOURCE_DIR}/Mode
 
 MAKE_SWIG_TARGET(OpenStudioModelHVAC model "${CMAKE_CURRENT_SOURCE_DIR}/ModelHVAC.i" "${${target_name}_swig_src}" ${swig_target_name} OpenStudioModelGeometry)
 
+MAKE_SWIG_TARGET(OpenStudioModelZoneHVAC model "${CMAKE_CURRENT_SOURCE_DIR}/ModelZoneHVAC.i" "${${target_name}_swig_src}" ${swig_target_name} OpenStudioModelHVAC)
+
+MAKE_SWIG_TARGET(OpenStudioModelSetpointManager model "${CMAKE_CURRENT_SOURCE_DIR}/ModelSetpointManager.i" "${${target_name}_swig_src}" ${swig_target_name} OpenStudioModelHVAC)
+
+MAKE_SWIG_TARGET(OpenStudioModelAvailabilityManager model "${CMAKE_CURRENT_SOURCE_DIR}/ModelAvailabilityManager.i" "${${target_name}_swig_src}" ${swig_target_name} OpenStudioModelHVAC)
+
+MAKE_SWIG_TARGET(OpenStudioModelPlantEquipmentOperationScheme model "${CMAKE_CURRENT_SOURCE_DIR}/ModelPlantEquipmentOperationScheme.i" "${${target_name}_swig_src}" ${swig_target_name} OpenStudioModelHVAC)
```
Comment (macumber) [inline openstudiocore/src/model/CMakeLists.txt]: @kbenne I think we may need to be more linear in our dependency graph here, I hate the impact on compile time though, we need to think this through more

Comment (kbenne) [reply openstudiocore/src/model/CMakeLists.txt]: Are you experiencing some trouble? I did not test it with Visual Studio. I don't think there is anything intrinsically wrong with having multiple targets depend on the same thing.

Comment (macumber) [reply openstudiocore/src/model/CMakeLists.txt]: I need to think through what this means in SWIG land.  Based on what I just went through for AFN in C# I expect there will be more work required to get this working nicely in C#.

**Comment by MingboPeng** [comment]
FYI https://github.com/NREL/OpenStudio/issues/3398 
"SWIGTYPE_p_openstudio__model__ZoneHVACTerminalUnitVariableRefrigerantFlow".

**Pull request merged**